packages/backend/src/links/entities/link.entity.ts

```diff
@@ -1,32 +1,52 @@
-import { IsString, IsNotEmpty, IsUrl, IsOptional, IsArray, ArrayMinSize, IsUUID } from 'class-validator'
+import { ApiHideProperty } from '@nestjs/swagger'
+import { ArrayMinSize, IsArray, IsNotEmpty, IsOptional, IsString, IsUUID, IsUrl } from 'class-validator'
 
 export class Link {
+  @ApiHideProperty()
```

Comment:
Im not sure anymore, if we need to hide this. Maybe showing it helps, when debugging.

Reply:
Opps I didn't spot this. It shouldn't be hidden, the OmitType will make sure it won't be present on the create dto, but otherwise it should be there. I fixed it real quick.